rslidar_driver/CMakeLists.txt

```diff
@@ -27,14 +27,14 @@ find_package(catkin REQUIRED COMPONENTS
   rospy
 	std_msgs
   genmsg
-  cv_bridge
```

Comment:
Is this also applicable change to the upstream?

Reply:
`cv_bridge` does not seem to be used so I guess we could open a PR upstream directly.

About the c++14, it looks like there is a PR opened already: https://github.com/RoboSense-LiDAR/ros_rslidar/pull/43/files.

Reply:
Thanks for sharing the info. That's good to know anyway.